# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLCrossProcessCommandQueue.cpp
# Commit: ccfa767dba64
# Full Hash: ccfa767dba644dc193e0246eb8e8ff3377e8b8a5
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Client-side bindings mirror for precise CC, and merge similar codepaths. r=handyman
#   
#   * Context loss using RAII
#   * Move Program reflection Client-side
#   
# ==============================================================================

diff -r 71c122ac0ca7 -r ccfa767dba64 dom/canvas/WebGLCrossProcessCommandQueue.cpp
--- a/dom/canvas/WebGLCrossProcessCommandQueue.cpp	Wed Jan 08 22:19:14 2020 +0000
+++ b/dom/canvas/WebGLCrossProcessCommandQueue.cpp	Wed Jan 08 22:19:16 2020 +0000
@@ -8,11 +8,6 @@
 
 namespace mozilla {
 
-HostWebGLCommandSink::HostWebGLCommandSink(
-    UniquePtr<Consumer>&& aConsumer,
-    UniquePtr<ProducerConsumerQueue>& aResponsePcq)
-    : SyncCommandSink(std::move(aConsumer), aResponsePcq) {}
-
 bool HostWebGLCommandSink::DispatchCommand(size_t command) {
   if (!mHostContext) {
     return false;
@@ -21,8 +16,4 @@
   return WebGLMethodDispatcher::DispatchCommand(command, *this, *mHostContext);
 }
 
-void HostWebGLCommandSink::ReportOOM() {
-  mHostContext->ReportOOMAndLoseContext();
-}
-
 }  // namespace mozilla